index.html

```diff
@@ -1,218 +1,165 @@
-				travisBadgeTemplate = $('#travis-badge-template').text(),
-				sections = [];
-
-			// Start building repo sections
-			repos.forEach(function (repo, index) {
-				// Replace placeholder with relevant data
-				var sectionTemplateStr,
-						travisTemplateStr,
-						repoId = repo.id,
-						travis_ci_build_branch = repo.travis_ci_build_branch || '';
-
-				// If this repo is Travis-CI enabled, produce a "badge", <a><img>
-				if (travis_ci_build_branch) {
-					travisTemplateStr =
-						travisBadgeTemplate
-						.replace(/\{repo-id\}/gi, repoId)
-						.replace(/\{travis_ci_build_branch\}/gi, travis_ci_build_branch);
-				}
-
-				// Construct a repo section
-				sectionTemplateStr = repoSectionTemplate
-					.replace(/\{repo-id\}/gi, repoId)
-					.replace(/\{travis-ci-build-status\}/gi, travisTemplateStr);
-
-				// Populate a hash for faster repo type lookup via repo id
-				repoTypeHash[repoId] = repo.type;
-
-				// Create a document fragment out of the template text and push it on the sections array
-				sections.push($(sectionTemplateStr));
-			});
-
-			// Append all sections to the DOM at once
-			$('#repo-section').append(sections);
-
-			// Fire a single HTTP request to the GitHub API to fetch stats (note that this does not support pagination yet)
-			$.getJSON('https://api.github.com/orgs/heremaps/repos?per_page=100')
-			.done(function (repo_data) {
-				repo_data.forEach(function (data, index) {
-					/* The GitHub API will return more repos than the ones we expose here.
-						 Target only existing repo sections.*/
-					var selector = data.name.replace(/\./g, '\\.')
-					var $target = $('#repo-info-' + selector),
-							repo_language,
-							travis_badge;
-					if ($target.length > 0) {
-						/* Check if we override the language ourselves by looking into the repoType hash.
-							 Fall-back to the value returned by the API; if that is null set to empty string
-						*/
-						repo_language = repoTypeHash[data.name] || data.language || '';
-
-						$target.find('[data-key="repo-type"]').text(repo_language);
-						$target.find('[data-key="repo-descr"]').text(data.description);
-						$target.find('[data-key="repo-lastUpdate"]').text(moment(data.updated_at).format('ll'));
-						$target.find('[data-key="fork-count"]').text(data.forks_count);
-						$target.find('[data-key="star-count"]').text(data.stargazers_count);
-					}
-				});
-			})
-			.fail(function () {
-				console.log(arguments, "Error fetching repository information from the GitHub API!");
-			});
-		</script>
-	</body>
-</html>
+<html>
+
+<head>
+   <title>HERE Technologies on Github</title>
+   <meta name="viewport" content="width=device-width,initial-scale=1">
+
+   <link rel="stylesheet" type="text/css" href="css/index.css">
+   <link rel="shortcut icon" href="resources/favicon.ico">
+
+   <script src="https://unpkg.com/three/build/three.min.js"></script>
+   <script src="https://unpkg.com/@here/harp.gl/dist/harp.js"></script>
```

Comment:
I would not link to latest here but rather to a fix version, because otherwise everytime we do a breaking API change that might affect this implementation, you need to remember to update the heremaps.github.io site.
If you fix to a particular version, you can control that.
And because we are growing right now very fast, we do break our styling json quite often.

Reply:
Added v0.2.2